roles/openshift_label_nodes/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: label the application nodes in the core cluster
+  shell: oc label node {{ item }} {{ label }}
```

Comment:
This will not work.  You need a full node domain.  I.e. 
`s/{{ item }}/{{ item }}.{{ clusterid }}.{{ dns_domain }}/
`